### PR TITLE
docs(readme): tune-up

### PR DESCRIPTION
**Pull request by mnaumanali94**
Updates readme.

**Checklist**

- [ ] Tests added / updated
- [x] Docs added / updated

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

**Additional context**

Hey @P0lip,

Made a few upgrades to the readme by adding visuals and a few subtle references to Stoplight.

Let me know what you think. Happy to make changes. 

P.S. I didn't create an issue for this. Should I?

### CODE REVIEW

**Comment by P0lip** [comment]
> P.S. I didn't create an issue for this. Should I?

No need to unless you want it for an internal (Stoplight) tracking

File: README.md [hunk 1]
```diff
@@ -1,28 +1,33 @@
-![Spectral logo](./docs/img/spectral-banner.png)
-
-[![CircleCI](https://img.shields.io/circleci/build/github/stoplightio/spectral/master)](https://circleci.com/gh/stoplightio/spectral) [![NPM Downloads](https://img.shields.io/npm/dw/@stoplight/spectral?color=blue)](https://www.npmjs.com/package/@stoplight/spectral) [![Treeware (Trees)](https://img.shields.io/treeware/trees/stoplightio/spectral)](https://plant.treeware.earth/stoplightio/spectral)
-
-A flexible JSON/YAML linter, with out of the box support for OpenAPI v2/v3 and AsyncAPI v2.
+<div align="center" width="1000">
+  <a href="https://stoplight.io/api-mocking?utm_source=github&utm_medium=spectral&utm_campaign=readme">
+    <img src="./docs/img/readme-header.svg" width="1000" alt="JSON/YAML linter with custom rulesets.">
```
Comment (P0lip) [inline README.md:3]: This new banner looks slick!

File: README.md [hunk 2]
```diff
@@ -1,28 +1,33 @@
-![Spectral logo](./docs/img/spectral-banner.png)
-
-[![CircleCI](https://img.shields.io/circleci/build/github/stoplightio/spectral/master)](https://circleci.com/gh/stoplightio/spectral) [![NPM Downloads](https://img.shields.io/npm/dw/@stoplight/spectral?color=blue)](https://www.npmjs.com/package/@stoplight/spectral) [![Treeware (Trees)](https://img.shields.io/treeware/trees/stoplightio/spectral)](https://plant.treeware.earth/stoplightio/spectral)
-
-A flexible JSON/YAML linter, with out of the box support for OpenAPI v2/v3 and AsyncAPI v2.
+<div align="center" width="1000">
+  <a href="https://stoplight.io/api-mocking?utm_source=github&utm_medium=spectral&utm_campaign=readme">
```
Comment (P0lip) [inline README.md:2]: Could we use markdown instead of HTML?
You can leave HTML for the `div` above (cause of `align`) and perhaps the image right below.

**Review by P0lip — COMMENTED** [review]
Looks good in general! I like the change.
I have only one remark re the usage of HTML, but other than this, it's good to go from my perspective.

File: README.md (hunk 2 above)
Comment (P0lip) [reply README.md:2]: oh, the URL is not correct. I suppose you don't want to point at mocking, do you?

Comment (P0lip) [inline README.md:2]:
```suggestion
  <a href="https://stoplight.io/api-governance?utm_source=github&utm_medium=spectral&utm_campaign=readme">
```

Comment (mnaumanali94) [reply README.md]: Thanks! Fixed this.

File: README.md (hunk 1 above)
Comment (mnaumanali94) [reply README.md]: Removed the html altogether. Our workspace doesn't deal very well with it.

**Comment by mnaumanali94** [comment]
@P0lip Thanks for the review. Can you give this another peak please?

Added an overview section because the first heading appears in the table of contents, also makes it easier to navigate within the GitHub readme.

**Review by philsturgeon — APPROVED** [review]
Amazing, thank you!

**Review by P0lip — APPROVED** [review]
Looks great, thanks!

**Review by P0lip — APPROVED** [review]

**Pull request merged**